Makefile

```diff
@@ -4,9 +4,9 @@
 #
 ################################################################################
 
-.PHONY : all minimal main solver check cantherm clean install decython documentation mopac_travis
+.PHONY : all minimal main solver check pycheck arkane clean install decython documentation mopac_travis
```

Comment:
(Outside the scope of this PR) Are our .PHONY targets up to date? Should things like `test` and `eg0` be in there?